pkg/apis/servicecatalog/testing/fuzzer.go

```diff
@@ -192,6 +193,11 @@ func FuzzerFor(t *testing.T, version schema.GroupVersion, src rand.Source) *fuzz
 			// Set the bytes field on the RawExtension
 			r.Raw = bytes
 		},
+		func(bs *servicecatalog.ServiceBrokerSpec, c fuzz.Continue) {
```

Comment:
I'm surprised that the default fuzzer didn't work for this field.  Do you try it?

Reply:
Think I did, but I'll pull it and try again.